### DISCUSSION TITLE
APK export?

### DISCUSSION THREAD

**Discussion by d4rken** [d4rken-org/sdmaid-se#362] [Ideas]
Does it make sense to add a feature that allows exporting APKs of installed apps?

I think usefulness of this is diminishing due to Google pushing app bundles which are not easy to install again either.
Google won't grant SD Maid "install permission", so while SD Maid could export the files, it would require a different app to reinstall.

An exported app bundle on one device might also not contain all the necessary files that another device might require, but that would be downloaded if installed via Google Play. App bundles != Universal APKs...

**Comment by Rokcy98** [comment]
I asked for this on xda, but, at the moment, maybe there are more important things to implement

**Comment by M-u-m-p-i-t-z** [comment]
If I test uninstall an apk I don't know, I can't add it back later?

Also there is no longer the option of autostart control, why?

**Comment by Oloooooooo** [comment]
+ 1 for exporting function (option), great idea :)

**Comment by frasca61** [comment]
The export of APK is very convenient to save the current version of an app before an update that may not work or be welcome. It is a function that I often use in SDMAID and that is not in this version

**Comment by d4rken** [comment]
#1026 will add export functionality
